tests/acceptance/project-conversations-test.js

```diff
@@ -64,112 +62,96 @@ test('Project admin can view list of conversations', function(assert) {
   server.create('conversation', { message:  message2, user, updatedAt: date2 });
   server.create('conversation', { message:  message3, user, updatedAt: date3 });
 
-  page.visit({
+  await page.visit({
     organization: project.organization.slug,
     project: project.slug
   });
 
-  andThen(() => {
-    assert.equal(page.conversations().count, 3, 'Conversations are rendered');
-    let renderedTimeStampOrder = [
-      page.conversations(0).updatedAt.text,
-      page.conversations(1).updatedAt.text,
-      page.conversations(2).updatedAt.text
-    ];
-
-    let expectedTimeStampOrder = [
-      'a few seconds ago',
-      'an hour ago',
-      '5 hours ago'
-    ];
-
-    assert.deepEqual(
-      renderedTimeStampOrder,
-      expectedTimeStampOrder,
-      'Items are rendered in correct order with properly formatted timestamps.'
-    );
-  });
+  assert.equal(page.conversations().count, 3, 'Conversations are rendered');
+  let renderedTimeStampOrder = [
+    page.conversations(0).updatedAt.text,
+    page.conversations(1).updatedAt.text,
+    page.conversations(2).updatedAt.text
+  ];
+
+  let expectedTimeStampOrder = [
+    'a few seconds ago',
+    'an hour ago',
+    '5 hours ago'
+  ];
+
+  assert.deepEqual(
+    renderedTimeStampOrder,
+    expectedTimeStampOrder,
+    'Items are rendered in correct order with properly formatted timestamps.'
+  );
 });
 
-test('Project admin can view single conversations', function(assert) {
+test('Project admin can view single conversations', async function(assert) {
   assert.expect(1);
 
   let { project, user } = server.create('project-user', { role: 'admin' });
   authenticateSession(this.application, { user_id: user.id });
 
   createConversations(3, project, user);
 
-  page.visit({
+  await page.visit({
     organization: project.organization.slug,
     project: project.slug
   });
 
-  andThen(() => {
-    page.conversations(1).click();
-  });
+  page.conversations(1).click();
```

Comment:
`await` here as well?  I thik it may b/c `click` is an async test helper.

https://github.com/san650/ember-cli-page-object/blob/master/addon/-private/properties/clickable.js#L68